controllers/packagebundlecontroller_controller.go

```diff
@@ -115,35 +115,48 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 		return withoutRequeue(result), nil
 	}
 
+	result.RequeueAfter = pbc.Spec.UpgradeCheckShortInterval.Duration
 	latestBundle, err := r.bundleManager.LatestBundle(ctx, pbc.Spec.Source.BaseRef())
 	if err != nil {
 		r.Log.Error(err, "Unable to get latest bundle")
-		if pbc.Status.State != api.BundleControllerStateDisconnected {
+		if pbc.Status.State == api.BundleControllerStateActive {
```

Comment:
Why do we want to change this condition? I'm aware that currently there are three possible states for bundle controller, and BundleControllerStateIgnored state won't reach here as it gets returned right before. But won't the original condition be much safer for future extension with more states added?

Reply:
Because we are now supporting a brand new PBC which would have the state `""`.  I do not want to change the state to disconnected if we haven't activated a bundle yet.